### ISSUE TITLE
Field views not valid xhtml

### ISSUE THREAD

**Issue by SpadXIII** [jonathangeiger/kohana-jelly#41]
Some of the field views are not xhtml valid: the input-tags aren't closed correctly.

```
// incorrect xhtml
<input type="text" name="<?php echo $name?>" value="<?php echo date($pretty_format, $value)?>" id="field-<?php echo $name?>">
// correct xhtml
<input type="text" name="<?php echo $name?>" value="<?php echo date($pretty_format, $value)?>" id="field-<?php echo $name?>" />
```

**Comment by banks** [comment]
Thanks. We'll take a look at this. Jon, how do you feel about using Form::input() where possible in the default views? That way we can auto add an empty $attr variable and people can then pass that through, effectively compromising between using views and the flexibility many wanted of the form helper.

I suggest we ensure $vars['attrs'] is always an empty array if not passed in `input()` and then in the views do something like.

```
<?php echo Form::input($name, $value, $attrs + array('default' => 'attribute value'));?>
```

I could then pass in other attributes if necessary:

```
$field->input(NULL, array('attrs' => array('class' => 'big_input')));
```

**Comment by SpadXIII** [comment]
It would indeed make sense to use the Form class where possible. It would at least guarantee that the produced html would be valid for the framework.

**Comment by banks** [comment]
I've implemented most of this. I just need to clean up relationship inheritance issues. I'll do that in the next day or so and commit this.

**Comment by jonathangeiger** [comment]
This is a fine compromise.

**Comment by dewos** [comment]
Nice!! Btw 'attrs' should be 'attributes'. I'm really not a fan of the 'slimy-name' variables.

**Comment by banks** [comment]
Updated field input templates to use Form helper. Closed by 0da43a3ab3cff03374c4aa343b591e46a7990cff.

**Comment by banks** [comment]
> 'attrs' should be 'attributes'

Yep, should at least be consistent with Form and HTML... I took this into account.
